.gitignore

```diff
@@ -40,3 +40,7 @@ testem.log
 # System Files
 .DS_Store
 Thumbs.db
+
+.htaccess
+config.json
+widgetbeheer-angular-deploy-config
```

Comment:
Nit: We should probably clean up all these unrelated ignores like `.DS_Store`, `.vscode`, `.idea` etc.